config/configuration.yml

```diff
@@ -21,15 +21,10 @@ default:
   email_delivery:
     delivery_method: :smtp
     smtp_settings:
-      enable_starttls_auto: <%= ENV['SMTP_ENABLE_STARTTLS_AUTO'] %>
+      enable_starttls_auto: true
```

Comment:
If this value is always `true`, `SMTP_ENABLE_STARTTLS_AUTO=false` line is not necessary in `.env.example`.